hs_modflow_modelinstance/receivers.py

```diff
@@ -173,68 +169,57 @@ def process_package_info(resource):
                         specified_head_boundary_packages=specified_head_boundary_package_list)
-                    continue
                 if p in modflow_models.uncouple(
                         modflow_models.BoundaryCondition.specifiedFluxBoundaryPackageChoices):
                     # create if does not exist, update if it does exist
                     specified_flux_boundary_packages_list.append(p)
                     create_or_update_from_package(
                         resource, modflow_models.BoundaryCondition,
                         specified_flux_boundary_packages=specified_flux_boundary_packages_list)
-                    continue
                 if p in modflow_models.uncouple(
                         modflow_models.BoundaryCondition.headDependentFluxBoundaryPackageChoices):
                     # create if does not exist, update if it does exist
                     head_dependent_flux_boundary_packages.append(p)
                     create_or_update_from_package(
                         resource, modflow_models.BoundaryCondition,
                         head_dependent_flux_boundary_packages=head_dependent_flux_boundary_packages)
-                    continue
                 # ModelCalibration
                 if p in modflow_models.uncouple(
                         modflow_models.ModelCalibration.observationProcessPackageChoices):
                     # create if does not exist, update if it does exist
                     create_or_update_from_package(resource, modflow_models.ModelCalibration,
                                                   observationProcessPackage=p)
-                    continue
                 # GeneralElements
                 if p in modflow_models.uncouple(
                         modflow_models.GeneralElements.modelSolverChoices):
                     # create if does not exist, update if it does exist
                     create_or_update_from_package(resource, modflow_models.GeneralElements,
                                                   modelSolver=p)
-                    continue
                 if p in modflow_models.uncouple(
                         modflow_models.GeneralElements.outputControlPackageChoices):
                     # create if does not exist, update if it does exist
                     output_control_package_list.append(p)
                     create_or_update_from_package(
                         resource, modflow_models.GeneralElements,
                         output_control_package=output_control_package_list)
-                    continue
                 if p in modflow_models.uncouple(
                         modflow_models.GeneralElements.subsidencePackageChoices):
                     # create if does not exist, update if it does exist
                     create_or_update_from_package(resource, modflow_models.GeneralElements,
                                                   subsidencePackage=p)
-                    continue

```

Comment:
Only this ```continue``` was not needed. I think you need all other ```continue``` above to avoid executing multiple ```if``` statements within each loop if you know if one ```if``` is true then other ```if```s can't be true.